router/router.go

```diff
@@ -171,6 +172,14 @@ func Load(middleware ...gin.HandlerFunc) http.Handler {
 		debugger.GET("/pprof/trace", debug.TraceHandler())
 	}
 
+	monitor := e.Group("/api/metrics")
+	{
+		monitor.GET("/prometheus",
+			session.MustAdmin(),
```

Comment:
Do we need authorization on that endpoint? That means, the app that fetches the metrics needs to be authorized? If authorization is desired, can we lower it at least to MustUser() for now?